sound/soc/intel/common/Makefile

```diff
@@ -9,7 +9,7 @@ snd-soc-acpi-intel-match-objs := soc-acpi-intel-byt-match.o soc-acpi-intel-cht-m
 	soc-acpi-intel-cml-match.o soc-acpi-intel-icl-match.o \
 	soc-acpi-intel-tgl-match.o soc-acpi-intel-ehl-match.o \
 	soc-acpi-intel-jsl-match.o soc-acpi-intel-adl-match.o \
-	soc-acpi-intel-hda-match.o \
+	soc-acpi-intel-hda-match.o soc-acpi-intel-mtl-match.o \
```

Comment:
> nit-pick: maybe move it up one line to be consistent with the never-ending list of platforms, and leave hda alone since it's a specific case?

ack